src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -626,6 +640,48 @@ public void ShowHotspotRequested_CounterIncremented(int previousCounter)
             telemetryRepositoryMock.Verify(x=> x.Save(), Times.Once);
         }
 
+        [TestMethod]
+        [DataRow(0)]
+        [DataRow(1)]
+        [DataRow(100)]
+        public void TaintIssueOpened_CounterIncremented(int previousCounter)
+        {
+            var telemetryData = new TelemetryData { Taint = new Taint { NumberOfOpenedIssues = previousCounter } };
+            telemetryRepositoryMock.Setup(x => x.Data).Returns(telemetryData);
+            var testSubject = CreateManager();
+
+            // Set up the telemetry mock after creating the manager, since the manager will
+            // may saved data on initial creation
```

Comment:
The comment needs to be tweaked: "... manager will may saved data..."

Reply:
Darn copy-paste.